docs/api/conventions.md

```diff
@@ -1307,27 +1252,23 @@ Another common case is when you've got nested routes and a child component has a
 
 Consider these routes:
 
-```
```

Comment:
@mcansh I think your plugin needs some work (probably a good idea to add tests for these), as it appears that's the one removing code block start/end (` ``` `).

Reply:
looks like we lost the code fencing, but markdown still renders it the same

Reply:
Fencing gets removed & 4 spaces are put into place, which should indeed give the same rendered result normally.

However, I think having the fences makes it more clear though.

Reply:
Thinking about this, this could be a (semi) `prettier` problem as well, as code blocks aren't formatted consistently, but I don't know what the expected behavior is for them.

Reply:
looks like fences are an option of remark - adding them back